ferrocene/ci/scripts/detect-conflict-markers.py

```diff
@@ -36,6 +36,9 @@
     "<<<<<<< HEAD //~ ERROR encountered diff marker",
     # Introduced in https://github.com/rust-lang/rust/pull/116712.
     ">>>>>>> 7a4f13c blah blah blah",
+    # Example inside rust-dev-guide subtree
+    "<<<<<<< HEAD\n Original code",
+    ">>>>>>> 8fbf656... Commit fixes 12345",
```

Comment:
So, the first exception you added doesn't do anything, as [the conflict marker checker](https://github.com/ferrocene/ferrocene/blob/5c27495e6d5ae4813edc08e3b7afe4c1f00e61a4/ferrocene/ci/scripts/detect-conflict-markers.py#L61) checks the file line by line.

Still, the script is detecting no conflict marker, in spite of the first exception not being valid, and that is expected! The script reports a conflict only if it finds the start, the middle *and* the end of a conflict marker. By only ignoring the end (`>>>>>>> 8fbf656... Commit fixes 12345`) you successfully ignore the whole conflict block, as the script can't find the end.

I'd remove the first exception and only keep the second one you added.